spec.bs

```diff
@@ -573,18 +575,23 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
   1. [=list/For each=] |component| in |auctionConfig|'s [=auction config/component auctions=],
     [=parallel queue/enqueue steps|enqueue the following steps=] to |queue|:
     1. Let |compWinner| be the result of [=generate and score bids=] with |component|,
-      |auctionConfig|, and |global|.
-    1. If |compWinner| is not failure or null, [=score and rank a bid=] with |auctionConfig|,
-      |compWinner|, |leadingBidInfo|, |desisionLogicScript|, and true.
+      |auctionConfig|, true, and |global|.
+    1. If |compWinner| is not failure or null:
+      1. Run [=score and rank a bid=] with |auctionConfig|, |compWinner|, |leadingBidInfo|,
+        |desisionLogicScript|, and true.
     1. Decrement |pendingComponentAuctions| by 1.
   1. Wait until |pendingComponentAuctions| is 0.
+  1. Let « |sellerSignals|, |reportResultBrowserSignals| » be the result of running
+    [=report result=] with |leadingBidInfo|.
+  1. Run [=report win=] with |leadingBidInfo|, |sellerSignals|, and |reportResultBrowserSignals|.

```

Comment:
1. top level seller's report result is called separately below, since it covers both the case of a non-SSP auction's reportResult, and top level auction's reportResult there. https://github.com/WICG/turtledove/pull/532/files#diff-6f5a1d8263b0b0c42e2716ba5750e3652e359532647ac934c1c70086ae3ceddaR655.
2. and 3. yes, will update in a later commit.

Reply:
> 1. top level seller's report result is called separately below, since it covers both the case of a non-SSP auction's reportResult, and top level auction's reportResult there. 

I don't think it should be called below (and I don't think it is due to the line 653 check) because we don't yet know if that component auction is the overall winner.

Reply:
yeah you're right. Was wrong about where the top level auction ends.